tests/processes/ProcessingTest.py

```diff
@@ -816,3 +818,50 @@ def test_autoapply_override(self):
                                       self.section, self.log_printer, {}, [],
                                       console_printer=self.console_printer)
         self.assertEqual(newres, [])
+
+
+class ESLintBear_apply_patches(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.section.append(Setting('files', '*.js, **/*.js, **/**/*.js'))
+        self.section.append(Setting('eslint_config', '.eslintrc'))
+        self.section.append(Setting('bears', 'ESLintBear'))
```

Comment:
This requires `ESLintBear` to be installed, right? I would like to ask you to write a custom bear instead so coala keeps independent from `coala-bears`, even for testing.